### PR TITLE
CIP-0018 | Adjust preamble and structure w.r.t CIP-0001

### LINKED ISSUE #642: CIP-0018 | Adjust preamble and structure w.r.t CIP-0001
Remediating this one out of numerical sequence in #389 TOC because of relevance to contemporary discussion in #628.

Fixed by #643.

### PR DESCRIPTION
**Pull request by rphair**
Fixes #642.

([edited version](https://github.com/rphair/CIPs/blob/remediate-0018/CIP-0018/README.md))

### CODE REVIEW

File: CIP-0018/README.md [hunk 1]
```diff
@@ -55,42 +56,42 @@ Key de-registrations work symmetrically and require that:
...
+### Acceptance Criteria
 
-The `minUTxOValue` is fixed by the protocol. It is defined as part of the Shelley genesis and can be updated via on-chain protocol updates. At the moment, this equals 1 Ada on the mainnet. 
+- [ ] There exists one or more reference implementations with appropriate testing illustrating the viability of this approach and specification.
 
-#### [2] Mangled Addresses
+### Implementation Plan
 
-An address is said mangled when it has a stake part, and the stake part isn't recognized as belonging to its associated wallet. That is, the payment part and the stake part appear to come from two different sources. This could be the case if the address has been purposely constructed in such a way (because the stake rights and funds are managed by separate entities), or because the stake part refers to a key hash which is no longer known of the wallet (because the associated key registration was rolled back).
+- [ ] Develop the proposed Reference Implementation as suggested when this CIP was originally published (see Discussion link in header for history).
```
Comment (Ryun1) [inline CIP-0018/README.md:89]:
```suggestion
- [ ] Update this proposal to account for the Conway Ledger era, which brings new types of certificates for registering stake keys.

- [ ] Develop the proposed Reference Implementation as suggested when this CIP was originally published (see Discussion link in header for history).
```

Not sure if we want to add a note or something like this? to signal to readers that this CIP should be updated for the Conway ledger era. 
As Conway changes how stake keys can be registered, as well as adding new roles to stake keys (governance rights), which I believe impacts this CIP.

**Review by Ryun1 — APPROVED** [review]
Great work Robert 🤠

See my one small comment, regardless I'm happy to approve.

File: CIP-0018/README.md (hunk 1 above)
Comment (rphair) [reply CIP-0018/README.md:89]: yes @Ryun1 I think it's proactive & helpful to include that in the plan.

**Review by Crypto2099 — APPROVED** [review]
LGTM

**Pull request merged**